.golangci.yml

```diff
@@ -41,7 +41,31 @@ linters-settings:
     ]
 
 issues:
+  exclude-use-default: false
   exclude:
+    # Add all default excluded issues except issues related to exported types/functions not having
```

Comment:
Where is this list of default excluded issues (and reasoning) from? I see similar snippets in other libraries (from HashiCorp, [for example](https://sourcegraph.com/github.com/hashicorp/raft/-/blob/.golangci-lint.yml))?

Reply:
The default excludes are listed in the [golangci-lint configuration page](https://golangci-lint.run/usage/configuration/#command-line-options) under "--exclude-use-default". I will include a link in the comment to clarify where the defaults are coming from.